tests/controller_tests/test_ingest.py

```diff
@@ -392,6 +393,25 @@ def test_ingest_endpoint_returns_validation_errors(test_client, example_data_mod
     assert "TabErrors" in validation_errors
 
 
+def test_ingest_endpoint_returns_uncaught_ingest_error(test_client, example_data_model_file, mocker):
+    """
+    Tests that, during ingest when an uncaught exception occurs, the endpoint returns the correct
+     500 response.
+    """
+
+    with mocker.patch("core.controllers.ingest.validate", side_effect=UnimplementedErrorMessageException()):
+        endpoint = "/ingest"
+        response = test_client.post(
+            endpoint,
+            data={
+                "excel_file": example_data_model_file,  # only passed to get passed the missing file check
```

Comment:
do we need to test the exc_info itself too, or just the summary message?

Reply:
good shout